client/sender-app/start/start.html

```diff
@@ -1,6 +1,15 @@
 <section class="container-fluid body-content">
   <div class="controls small-center">
-    <img ng-show="!vm.connectionStatus" ng-click="vm.connect()" src="content/ic_cast_grey_24dp.png" alt="chromecast icon">
+    <img ng-show="!vm.connectionStatus" ng-click="vm.hasExtension && vm.onChrome && vm.connect()" src="content/ic_cast_grey_24dp.png" alt="chromecast icon">
+    <div class="modal-content" ng-show="!vm.onChrome" >
+      <p class="start-modal">Ummm, this is awkward...</p>
+      <p class="start-modal">You'll need Google Chrome to play this on a laptop/desktop...  <a href="https://www.google.com/chrome/browser/desktop/index.html" target="_blank">Download Chrome</a> to get it.</p>
+      <p class="start-modal">Once you've downloaded Chrome head over to the <a href="https://chrome.google.com/webstore/detail/google-cast/boadgeojelhgndaghljhdicfkmllpafd?hl=en" target="_blank">Chrome Web Store</a> and download the Google Cast Extension, you'll need that too!</p>
+    </div>
```

Comment:
Do we want to have the message account for the fact that many people will have chrome? Could just change it to "Download Chrome if you don't already have it or fire it up if you do".

Or maybe at the end a message in parens: "(Or maybe you already have Chrome installed. In that case, fire it up. You'll need the Chromecast extension. If you don't have it, we'll give you a link once you're over in Chrome)"